src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-block: 32px 16px;
+  padding-inline: 16px;
+
+  &__image {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+      background-image: url(/src/images/imac.jpeg);
```

Comment:
Avoid using absolute paths for images. It's better to use a relative path so that the image remains accessible even if the project structure changes or when the project is moved to a different environment.